src/S3Policy.js

```diff
@@ -51,12 +53,12 @@ const getDate = () => {
  */
 const getExpirationDate = () => {
   return new Date(
-    (new Date).getTime() + FIVE_MINUTES
+    (new Date).getTime() + FIVE_MINUTES + (Math.abs(options.timeDelta) || 0)
```

Comment:
I may be missing something, but wouldn't `options` be a reference error here? I'm not seeing it passed in to this function.